core/base/src/TColor.cxx

```diff
@@ -1980,6 +1980,27 @@ Int_t TColor::GetColor(Int_t r, Int_t g, Int_t b, Float_t a)
    return color->GetNumber();
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Static method returning color number for a color specified by name.
```

Comment:
Where is the full list of colours which names I can choose from? I believe it is `TROOT::GetListOfColors`? Maybe a comment about it in the doc?

Reply:
To actually get the full list of color names one must do:

```cpp
const auto *colors = gROOT->GetListOfColors();
for (const auto *color:  TRangeDynCast<TColor>(colors)){
    if (!color) continue;
    std::cout << color->GetName() << "\n";
}
```

A bit too tedious probably, would be best to have it as a table somewhere and linked here in the doc

Reply:
See ListColors

Reply:
If with that you mean that "a user should see TColor::ListColors to see which color names they could use as input argument to TColor::GetColorByName", then could that be added to the documentation of this function?

Reply:
I will add it. I am trying to understand your new version of GetColorByname.